src/components/verification-grid/css/style.css

```diff
@@ -0,0 +1,93 @@
+#highlight-box {
+  position: absolute;
+  display: none;
+  top: 0px;
+  left: 0px;
+  width: 0px;
+  height: 0px;
+  background-color: var(--oe-selected-color);
+  border: solid 2px var(--oe-border-color);
+  border-radius: 1rem;
+  opacity: 0.3;
+  z-index: 5;
+
+  /* TODO: this isn't working, we should probably bubble events */
+  pointer-events: none;
+}
+
+/* TODO: fix placement */
+.statistics-section {
+  position: relative;
+  display: inline-block;
+  width: fit-content;
+  padding: var(--oe-spacing);
+}
+
+.verification-container {
+  background-color: var(--oe-background-color);
+  height: 100%;
+}
+
+.verification-grid {
+  display: flex;
+  flex-wrap: wrap;
+  user-select: none;
+  justify-content: center;
+  align-items: stretch;
+  gap: var(--oe-spacing);
+  margin: var(--oe-spacing);
```

Comment:
Gap effects the margin of content, while margin effects the margin of the current element.
Similar to `justify-items` vs `justify-self`

I use margin here so that the verification grid isn't flush with the edge of the screen (just slightly offset)

![image](https://github.com/user-attachments/assets/e9b8c033-dd81-41f2-9feb-4b5ac8d284ee)

_FireFox devtools Legend_

- Dashed background: gap
- Yellow background: margin

Reply:
yeah so wrapper with padding and gap would achieve the same affect?

Reply:
Opened an issue to tackle the spacing inconsistencies between grid tiles and the start/end of the verification-grid component

https://github.com/ecoacoustics/web-components/issues/88